pact-data-model/src/lib.rs

```diff
@@ -554,6 +565,132 @@ impl From<String> for SpecVersionString {
     }
 }
 
+impl JsonSchema for GeographicScope {
+    fn schema_name() -> String {
+        "GeographicScope".to_string()
+    }
+
+    fn json_schema(gen: &mut schemars::gen::SchemaGenerator) -> Schema {
+        let regional_or_subregion = {
+            let mut obj = ObjectValidation::default();
+            obj.required
+                .insert("geographyRegionOrSubregion".to_string());
```

Comment:
I would recommend you to refactor all occurances of `"geoegraphyRegionOrSubregion"`, `"geographyCountry"` etc. into constants (or a string that you duplicate in each case)

Reply:
Sounds good to me!